xml-binding/2.3/_index.md

```diff
@@ -0,0 +1,37 @@
+---
+title: "Jakarta XML Binding Specification 2.3"
+date: 2019-07-22T17:54:14+0200
+summary: "Initial release of Jakarta XML Binding"
+---
+The Jakarta XML Binding provides an API and tools that automate the mapping
+between XML documents and Java objects.
+
+* [Jakarta XML Binding 2.3 Specification Document](./jakarta.xml.bind-spec-2.3.pdf) (PDF)
```

Comment:
Per the comments on the other related PRs, the name of the Spec Doc should be xml-binding-spec-2.3.pdf per our [naming doc](https://github.com/jakartaee/specification-committee/blob/master/names.adoc).